src/audio/multiband_drc/multiband_drc.c

```diff
@@ -64,20 +64,25 @@ static void multiband_drc_reset_state(struct multiband_drc_state *state)
 		multiband_drc_iir_reset_state_ch(&state->deemphasis[i]);
 }
 
-static int multiband_drc_eq_init_coef_ch(struct sof_eq_iir_biquad *coef,
+static int multiband_drc_eq_init_coef_ch(struct comp_dev *dev, struct sof_eq_iir_biquad *coef,
 					 struct iir_state_df2t *eq)
 {
 	int ret;
 
 	eq->coef = rzalloc(SOF_MEM_ZONE_RUNTIME, 0, SOF_MEM_CAPS_RAM,
 			   sizeof(struct sof_eq_iir_biquad) * SOF_EMP_DEEMP_BIQUADS);
-	if (!eq->coef)
+	if (!eq->coef) {
+		comp_err(dev, "Failed to allocate EQ coefficients");
```

Comment:
the return code is logged by the caller which will point to this line, this is not needed